temporal-server/rockcraft.yaml

```
@@ -0,0 +1,30 @@
+name: temporal-server
+# see https://documentation.ubuntu.com/rockcraft/en/1.11.0/explanation/bases/
+# for more information about bases and using 'bare' bases for chiselled rocks
+base: ubuntu@24.04 # the base environment for this rock
+version: '1.23' # just for humans. Semantic versioning is recommended
```

```suggestion
version: '1.23.1' # just for humans. Semantic versioning is recommended
```

I'd suggest using the same version as the upstream project. Specifying the patch version reduces the risk of a silent dependency update in the charm potentially causing issues.